common/src/app/common/features.cljc

```diff
@@ -86,7 +88,8 @@
         "fdata/shape-data-type"
         "plugins/runtime"
         "design-tokens/v1"
-        "text-editor/v2"}
+        "text-editor/v2"
+        "render-wasm/v1"}
```

Comment:
Yo don't need to add here the flag, if it is already on `frontend-only-features` because these two sets are merged.

Reply:
But it's the same situation that we had with text-editor/v2 and we needed to add it too